applications/nrf5340_audio/src/bluetooth/bt_stream/unicast/unicast_server.c

```diff
@@ -649,6 +649,12 @@ int unicast_server_enable(le_audio_receive_cb recv_cb, enum bt_audio_location lo
 {
 	int ret;
 	static bool initialized;
+	const struct bt_pacs_register_param pacs_param = {
+		.snk_pac = true,
+		.snk_loc = true,
+		.src_pac = true,
+		.src_loc = true,
+	};
```

Comment:
This may depends on CONFIG_BT_AUDIO_TX. @Thalley: Should sink and source always be exposed like this? Even if we only have sink? I see that we set available context supported to `BT_AUDIO_CONTEXT_TYPE_PROHIBITED` in the case where we only have sink